wooting-macro-backend/src/lib.rs

```diff
@@ -319,13 +319,15 @@ fn check_macro_execution_efficiently(
                         if pressed_events == *data {
                             debug!("MATCHED MACRO singlekey: {:#?}", pressed_events);
 
-                            // ? Kinda works for now but needs to be improved. Disabled for now as its more of a regression than a fix.
+                            let channel_clone_lift = channel_sender.clone();
+                            let channel_clone_execute = channel_sender.clone();
+                            let macro_clone_execute = macros.clone();
+                            let pressed_data_clone = data.clone();
 
-                            let channel_clone = channel_sender.clone();
-                            let macro_clone = macros.clone();
+                            plugin::util::lift_keys(pressed_data_clone, channel_clone_lift);
 
```

Comment:
Do these really need to be cloned to be given to lift_keys? I think now you should just be able to pass them by reference to be used briefly